sphinx/ext/apidoc.py

```diff
@@ -0,0 +1,439 @@
+# -*- coding: utf-8 -*-
```

Comment:
I'm not able to do anything about this. Git does file rename detection automatically (`git [mv]` might well be an alias for `git cp [src] [dst]` and `git rm [src]`), and its heuristics seem to be failing us here.